app/controllers/cluster_infos_controller.rb

```diff
@@ -14,9 +14,14 @@ def show
         v.map!{|array_value| ERB::Util.h(array_value)} : v.kind_of?(Hash)?
             v.update(v){ |k_s,v_s| ERB::Util.h(v_s)} : ERB::Util.h(v)}
 
+    if params[:stand_alone] == 'false' || params[:stand_alone].nil?
```

Comment:
layout_value = not (params[:stand_alone] == 'false' || params[:stand_alone].nil?)